nav2_bringup/bringup/launch/tb3_simulation_launch.py

```diff
@@ -149,16 +163,29 @@ def generate_launch_description():
         name='robot_state_publisher',
         namespace=namespace,
         output='screen',
-        parameters=[{'use_sim_time': use_sim_time}],
-        remappings=remappings,
-        arguments=[urdf])
+        parameters=[{'use_sim_time': use_sim_time,
+                     'robot_description': robot_description}],
+        remappings=remappings)
+
+    robot_model_sdf = os.path.join(bringup_dir, 'worlds', 'waffle.model')
+
+    start_gazebo_spawner_cmd = Node(
+        package='gazebo_ros',
```

Comment:
Sweet! If we can actually get rid of the nav2_gazebo_spawner package now? If so, feel free to delete it in this PR now!